test/cpp_extensions/open_registration_extension/torch_openreg/setup.py

```diff
@@ -85,6 +85,7 @@ def main():
         cmdclass={
             "clean": BuildClean,  # type: ignore[misc]
         },
+        include_package_data=False,
```

Comment:
Thank you, Great.

`torch_openreg` supports direct installation and in-place installation (`editable`), can you help support source code distribution (`bdsit`)?